README.md

```diff
@@ -92,7 +92,7 @@ sudo apt-get install libjsoncpp-dev
 
 Run Mysys2 shell(but not PowerShell) to install external dependencies.
 
-```bash
```

Comment:
This highlight is good for bash shell script but not single line command...It looks strange sometimes because it highlight sudo cd. But alright, we can keep this if you prefer. I rebase this commit.